pkgs/applications/graphics/pixinsight/default.nix

```diff
@@ -128,7 +185,7 @@ stdenv.mkDerivation rec {
     license = licenses.unfree;
     platforms = [ "x86_64-linux" ];
     maintainers = [ maintainers.sheepforce ];
-    hydraPlatforms = [];
+    hydraPlatforms = [ ];
```

Comment:
Why is this here?

Reply:
Has been added some time ago by someone to avoid Hydra attempts of building this package and subsequently fail. This is what `lib.meta.dontDistribute` does and is correct for this package.
https://github.com/NixOS/nixpkgs/blob/938aa157bbd6e3c6fd7dcb77998b1f92c2ad1631/lib/meta.nix#L26

Reply:
Hydra won't even try to build `unfree` packages, FYI.

Reply:
See here for the reason: https://github.com/NixOS/nixpkgs/pull/190511